### DISCUSSION TITLE
Theoretically, would it be possible to make a custom Debian Linux distro for running Linux applications in the browser?

### DISCUSSION THREAD

**Discussion by hello-smile6** [copy/v86#497] [Ideas]
Would it be possible to make a distro with a custom X window server that, instead of displaying normally, would send the data for each window to v86, which would somehow open it in another window and let the application be used in that window? I would like to be able to run Linux software that way, kind of like WSL or Chromium OS's Linux crostini VM.

**Comment by Pixelsuft** [comment]
emulator.read_memory()
emulator.write_memory()
????

Create guest program that scanning all windows. (I think that all values will be loaded to memory) (for example, convert all values to string).
Then scanning all memory find offset of programs and save state (find this string (in bytes), when saving state offset will be same).

**Reply by hello-smile6** [reply]
Can you make a POC?

**Reply by Pixelsuft** [reply]
I'm noob (or just lazy) for making programs for linux

**Reply by hello-smile6** [reply]
> I'm noob (or just lazy) for making programs for linux

Please?
